snapshots/src/clob-liq.ts

```diff
@@ -0,0 +1,108 @@
+import { MapOfCount } from "./interfaces";
+import "cross-fetch/polyfill"; // polyfill vs ponyfill
+import { POLY_INTL_ID } from "./constants";
+
+export interface rewardsInt {
+    epoch: number;
+    market: string;
+    maker_address: string;
+    qfinal: string;
+}
+
+export interface makerScore {
+    maker_address: string;
+    qfinal: string;
+}
+
+export const getMarketsIncludedInEpoch = async (
+    clobUrl: string,
+    epoch: number,
+): Promise<string[]> => {
+    let marketsList: string[] = [];
+    await fetch(
+        `${clobUrl}/markets-included-in-epoch?epoch=${epoch.toString()}`,
+        {
+            method: "GET",
+            headers: {
+                POLY_INTL_ID: POLY_INTL_ID,
+            },
+        },
+    )
+        .then((response) => response.json())
+        .then((data) => {
+            marketsList = data["markets"].map((x) => String(x));
+        })
+        .catch((error) => {
+            console.error("Error:", error);
+            return "error";
+        });
+    return marketsList;
+};
+
+export const getTradersInEpoch = async (
+    clobUrl: string,
+    epoch: number,
+    marketsList: string[],
+): Promise<Set<string>> => {
+    let makers = new Set<string>();
+    for (var market of marketsList) {
+        const data = await fetch(
+            `${clobUrl}/liquidity-rewards-by-epoch?epoch=${epoch.toString()}&market=${market}`,
+            {
+                method: "GET",
+                headers: {
+                    POLY_INTL_ID: POLY_INTL_ID,
+                },
+            },
+        );
+
+        const buff = await data.arrayBuffer().then(Buffer.from);
+
+        const rewards: rewardsInt[] = JSON.parse(buff.toString())["rewards"];
+        for (var reward of rewards) {
+            makers.add(reward.maker_address);
+        }
+    }
+    return makers;
+};
+
+export const getLiquidtyRewardsForMakers = async (
+    clobUrl: string,
+    epoch: number,
+    makerList: string[],
+    feeTokenSupply: number,
+): Promise<MapOfCount> => {
+    let scoreMapping = {};
+    for (var maker of makerList) {
```

Comment:
nit: could use `const maker` and `const scoreMapping` since they aren't reassigned.